migrations/2023-06-30-111745_instance_alive_check/up.sql

```diff
@@ -0,0 +1 @@
+alter table site add column is_alive bool not null default true;
```

Comment:
What about `alive_check_fails_count integer default 0` instead of just a boolean? 

i.e `is_alive = alive_check_fails_count == 0`.

That way, an exponential backoff could be added later to the is_alive check, for example:

* one scheduled task running once per hour which retries all sites `where alive_check_fails_count = 1`
* one scheduled task running once per day which retries `where alive_check_fails_count < 5`
* one scheduled task running once per week which retries `where alive_check_fails_count >= 5`